Maple2.Server.Game/Util/ItemStatsCalculator.cs

```diff
@@ -157,6 +137,64 @@ public sealed class ItemStatsCalculator {
         return true;
     }
 
+    // Used to calculate the default constant attributes for a given item.
+    private bool GetConstantOption(Item item, int job, ItemOptionPickTable.Option? pick, [NotNullWhen(true)] out ItemStats.Option? option) {
```

Comment:
can you explain this function, I'm not able to deduce its rules

Reply:
The function will get all the stats defined in the ItemOptionConstantTable first.

Constant stats only use the ItemOptionPick table if their LevelFactor is over 50.
IF there were no stats in the Constant Table, it'll just run the Pick Table and lua functions as is. However, if there was a table, it'll feed the stat value into the lua function. It modifies the value a bit. I don't think ALL the lua functions make use of the stat value.